site/docs/v1/tech/identity-providers/nintendo.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: doc
+title: Nintendo Identity Provider
+description: Login with Nintendo using the Nintendo Identity Provider
+---
+
+:sectnumlevels: 0
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.36.0.
+====
+
+== Overview
+
+This section will cover how to add a Login with Nintendo button to FusionAuth. Below is an example login page with the Nintendo Identity Provider enabled.
+
+* <<Create a Nintendo Developer Account>>
+* <<Create a Nintendo Identity Provider>>
+* <<Building Your Own Integration>>
+
+{empty} +
+
+Once you have completed this configuration you will be able to enable the Nintendo login button for one or more FusionAuth Applications. Below is an example login page with the Nintendo Identity Provider enabled.
+
+image::identity-providers/login-nintendo.png[Nintendo Login]
+
+== Create a Nintendo Developer Account
+
+Follow instructions for obtaining a `Client id` and `Client secret` from the official Nintendo documentation.
```

Comment:
Follow the naming of the field names, this should be `Client Id`.

Reply:
Since the heading says "create a Nintendo Developer Account" it seems we should have a bit more here even if it is a sentence that says 

> Log into XYZ and create a developer account and enable the developer account for Login with Nintendo, and collect the Client Id and Client secret. Contact Nintendo Developer support for additional information. 

Or something simple like that.

Reply:
I took a shot. There is not much info available though.